lib/ansible/plugins/inventory/aws_ec2.py

```diff
@@ -17,13 +17,13 @@
         - aws_credentials
     description:
         - Get inventory hosts from Amazon Web Services EC2.
-        - Uses a YAML configuration file that ends with aws_ec2.(yml|yaml).
+        - Uses a YAML configuration file that ends with C(aws_ec2.(yml|yaml)).
     notes:
         - If no credentials are provided and the control node has an associated IAM instance profile then the
           role will be used for authentication.
     options:
         plugin:
-            description: token that ensures this is a source file for the 'aws_ec2' plugin.
+            description: Token that ensures this is a source file for the M(aws_ec2) plugin.
```

Comment:
The link here does not get built - if it did, it would be self-referential (from the current page to the current page), so not very useful as a link. I'd suggest code syntax instead:  
```suggestion
            description: Token that ensures this is a source file for the C(aws_ec2) plugin.
```

Reply:
Otherwise this PR looks good!

Reply:
wouldn't it actually try to point to (non existing) module of the same name?

Reply:
@bcoca yes, it would - but it's not worth fixing that underlying issue, since we don't really need a link there at all

Reply:
I have removed the plugin name from the description completely. I think it's not necessary there becase the `aws_ec2` is the only choice of the option anyway.